tests/APITests/APITest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Test\APITests;
+
+use App\Database\ConnectionManager;
+use App\Database\QueryBuilder;
+use App\Database\QueryWriterFactory;
+use App\Mapper\WordMapper;
+use App\Model\Word;
+use App\Repository\WordRepository;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+
+class APITest extends TestCase
+{
+    private static \PDO $connection;
+
+    public static function setUpBeforeClass(): void
+    {
+        self::$connection = ConnectionManager::getConnection();
+    }
+
+    public function tearDown(): void
```

Comment:
any reason to loosen visibility for method? (TestCase in phpunit has this method as private)